### PR TITLE
Rider support added

### LINKED ISSUE #1012: Rider
### Bug Description

I can install mirrord over the plugins in Rider, but it the icon isnt showing up like in Pycharm 

### Steps to Reproduce

1. Install mirrord plugin in Rider

### Backtrace

_No response_

### Relevant Logs

_No response_

### Your operating system and version

Linux

### Local process

no local process

### Local process version

_No response_

### Additional Info

_No response_

### PR DESCRIPTION
**Pull request by Razz4780**
Resolves #1012

### CODE REVIEW

File: intellij-ext/modules/products/rider/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+tasks {
+    buildSearchableOptions {
+        enabled = false
+    }
+}
+
+intellij {
+    version.set(properties("platformVersion"))
+    type.set("RD")
```
Comment (Razz4780) [inline intellij-ext/modules/products/rider/build.gradle.kts:21]: I have no idea why this line is required, as nothing similar happens in build files for other projects. It costed me a lot of sanity points.

**Review by infiniteregrets — APPROVED** [review]
LGTM overall, not sure about the version changes made, just make sure that doesn't break anything else

**Comment by aviramha** [comment]
Maybe add ignore for missing dependencies for now as you tested manually.
https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#tasks-runpluginverifier

`failureLevel﻿ `

**Comment by Razz4780** [comment]
> Maybe add ignore for missing dependencies for now as you tested manually. https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#tasks-runpluginverifier
> 
> `failureLevel﻿ `

Sure, can do. I also issued a ticket on youtrack, so hopefully it's not for a long time

**Comment by aviramha** [comment]
> > Maybe add ignore for missing dependencies for now as you tested manually. [plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#tasks-runpluginverifier](https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#tasks-runpluginverifier)
> > `failureLevel﻿ `
> 
> Sure, can do. I also issued a ticket on youtrack, so hopefully it's not for a long time

Great! I'd link the issue here so we can cross-track it.

**Comment by Razz4780** [comment]
> > > Maybe add ignore for missing dependencies for now as you tested manually. [plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#tasks-runpluginverifier](https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#tasks-runpluginverifier)
> > > `failureLevel﻿ `
> > 
> > 
> > Sure, can do. I also issued a ticket on youtrack, so hopefully it's not for a long time
> 
> Great! I'd link the issue here so we can cross-track it.

https://youtrack.jetbrains.com/issue/MP-5535

**Pull request merged**